root/components/RelationshipsTable.js

```diff
@@ -195,21 +181,71 @@ component RelationshipsTable(
 
   const pagedRelationshipGroups = entity.paged_relationship_groups;
   if (pagedLinkTypeGroup) {
+    const sourceType = entity.entityType;
+    const targetType = pagedLinkTypeGroup.relationships[0].target_type;
```

Comment:
I believe it's possible for there to be no relationships, which would throw an exception here. See e.g. https://github.com/metabrainz/musicbrainz-server/blob/afa5006/lib/MusicBrainz/Server/Controller/Role/Load.pm#L127

We could pass the `link_type` there too (alongside `link_type_id`), which will put it in `linkedEntities` for access here. So the target type would be something like `pagedLinkTypeGroup.backward ? linkType.entity_type0 : linkType.entity_type1` in that case.

Reply:
Hmm. If there's no rels, then we don't care about the table at all, do we? Can't we just then do `if (pagedLinkTypeGroup && pagedLinkTypeGroup.relationships.length > 0)`?